datafusion/src/sql/planner.rs

```diff
@@ -2959,6 +2990,36 @@ mod tests {
         quick_test(sql, expected);
     }
 
+    #[test]
+    fn select_order_by_index() {
+        let sql = "SELECT id FROM person ORDER BY 1";
+        let expected = "Sort: #person.id ASC NULLS LAST\
+                        \n  Projection: #person.id\
```

Comment:
It is better to add more complex case:
```
select .... from table order by 1,3; // with multi column
```
```
select * from table order by 1,3
```

Reply:
You can add more tests after https://github.com/apache/arrow-datafusion/issues/1372 merged.

Reply:
Sure I can add more